src/app/content/studyGuides/components/__snapshots__/StudyGuides.spec.tsx.snap

```diff
@@ -71,7 +71,7 @@ exports[`StudyGuides properly display summary highlights 1`] = `
             className="summary-highlight-content StudyGuidesListElement__HighlightContent-sc-5qgyda-2 jwHLiF"
             dangerouslySetInnerHTML={
               Object {
-                "__html": undefined,
+                "__html": "undefined",
```

Comment:
Should this be changed? It renders a `"undefined"` string but normally the `highlightedContent` can't be `undefined`

Reply:
I saw that and I think it doesn't really matter? I could change mock to contain something which will render this mocked value

Reply:
I don't think it matters either, just looks bit strange. There is probably nothing wrong with adding data to mocks that should be there